gstd/src/msg/encoded.rs

```diff
@@ -58,8 +59,8 @@ use scale_info::scale::{Decode, Encode};
 ///
 /// - [`load_bytes`](super::load_bytes) function returns a payload as a byte
 ///   vector.
-pub fn load<D: Decode>() -> Result<D> {
-    super::basic::with_read_on_stack(|read_result: Result<&mut [u8]>| -> Result<D> {
+pub fn load<D: Decode>() -> GstdResult<D> {
```

Comment:
crate::Result<D> looks better

Reply:
ok, but it will look like `crate::errors::Result`